index.html

```diff
@@ -370,7 +371,7 @@ <h2>Server Push (HTTP/2)</h2>
   Link: &lt;/app/style.css&gt;; rel=preload; as=style; nopush
   Link: &lt;/app/script.js&gt;; rel=preload; as=script
 </pre>
-      <p class="note">The above example indicates to an HTTP/2 push capable
+      <p class="note">The above example indicates to an HTTP/2 and HTTP/3 push capable
```

Comment:
Maybe replace "HTTP/2 and HTTP/3 push" with "Server Push"

Reply:
Updated